### ISSUE TITLE
[BUG] 11.3.1 iPad Pro 10" - App Store crashing

### ISSUE THREAD

**Issue by p0part** [eni9889/App-Admin-Issues#25]
I have this crash ever since the first beta/test release and is still present in the alpha release. When I try to downgrade a app in the app store, it either doesn't show the available versions or when it does, it crashes the app store without result.

I have attached the crashlog. 
[appadmin crash.txt](https://github.com/eni9889/App-Admin-Issues/files/2221009/appadmin.crash.txt)

**Comment by eni9889** [comment]
Does this happen every time?

**Comment by p0part** [comment]
Yes, every time I try to downgrade the app version (which does work perfectly on my iPhone X 11.2.2) it crashes. I have to respring in order to see the App Admin functions.

**Comment by eni9889** [comment]
Sadly I don't have an iPad Pro so I can't test it

**Comment by p0part** [comment]
Bummer! Even with the latest beta 94 it crashes. Also tried with all my other tweaks disabled.
